src/components/widgets/SideBarListItem.tsx

```diff
@@ -63,36 +77,40 @@ const useStyles = makeStyles((theme: ThemeType) => ({
 }))
 
 export interface SideBarListItemProps {
-
   isOpen: boolean
   isActive: boolean
   onClick: Function
   children: React.ReactNode
   variant?: 'light' | 'dark'
+  styleProps?: string
+  inStudyBuilder?: boolean
 }
 
 const SideBarListItem: React.FunctionComponent<SideBarListItemProps> = ({
-
   isOpen,
   isActive,
   onClick,
   children,
-  variant='light'
+  variant = 'light',
+  styleProps,
+  inStudyBuilder,
 }: SideBarListItemProps) => {
   const classes = useStyles()
   return (
     <li
-  
-      className={clsx(classes.listItem, {
-        [classes.listItemDark]: variant === 'dark', 
-        [classes.listItemActive]: isActive,
-        [classes.listItemCollapsed]: !isOpen,
-      })}
+      className={clsx(
```

Comment:
let's use a consistent way to define classes here. This can be written as 
```
  className={clsx({
        [classes.listItem]: !inStudyBuilder,
        [classes.studyBuilderListItem]: inStudyBuilder,
        [classes.listItemDark]: variant === 'dark',
        [classes.listItemActive]: isActive,
        [classes.listItemCollapsed]: !isOpen,
      })}
`